src-java/rule-manager/rule-manager-api/src/main/java/org/openkilda/ProtoConstants.java

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda;
+
+public final class ProtoConstants {
+
+    public static final class EthType {
+        public static final long ETH_TYPE_IPv4 = 2048;
+    }
+
+    public static final class IpProto {
+        public static final long UDP_IP_PROTO = 17;
+    }
+
+    public static final class Mask {
+        public static final long NO_MASK = -1;
+    }
+
+    public static final class PortNumber {
+        public static final int IN_PORT = -8;
```

Comment:
Bad idea. You become tied to OF constants. Need some wrapper to represent port number. It should contain enum field to represent reserved/special OF-ports and int field to represent generic port number (just raw thought, defenetly can be something better).

Reply:
Reworked.